examples/lang/consul1.mcl

```diff
@@ -0,0 +1,4 @@
+consul:kv "love" {
+	key   => "mgmt/love",
```

Comment:
Don't keys need to start with a `/` (slash?) If so shouldn't we validate them in Validate. Maybe we don't want to allow \0 or some other chars too?

Reply:
It does not, and https://www.consul.io/docs/agent/kv.html : `Keys, like objects are not restricted by type and can include any character.`

Reply:
everything is allowed